src/routes/(app)/+layout.svelte

```diff
@@ -55,6 +55,103 @@
       height: 100%;
       min-height: 100%;
       background-color: #f6f8fa;
+
+    }
+
+    :root {
+      /**
+        Custom properties to be defined(only surfacing what we have in our tokens.cjs)
+      */
+
+      /** colors*/
+      --tw-colors-focus-default-hs: 10, 24%;
+      --tw-colors-focus-default-l: 50%;
+
+      --tw-colors-background-dark-hs: 5, 30%;
+      --tw-colors-background-dark-l: 50%;
+      --tw-colors-background-light-hs:30, 100%;
+      --tw-colors-background-light-l: 50%;
+
+      --tw-colors-body-dark-hs: 122, 50%;
+      --tw-colors-body-dark-l: 50%;
+      --tw-colors-body-light-hs: 98, 18%;
+      --tw-colors-body-light-l: 50%;
+
+      --tw-colors-error-dark-hs: 0, 70%;
+      --tw-colors-error-dark-l: 50%;
+      --tw-colors-error-light-hs: 10, 100%;
+      --tw-colors-error-light-l: 50%;
+
+      --tw-colors-header-dark-hs: 33, 83%;
+      --tw-colors-header-dark-l: 50%;
+      --tw-colors-header-light-hs: 299, 92%;
+      --tw-colors-header-light-l: 50%;
+
+      --tw-colors-label-dark-hs: 98, 80%;
+      --tw-colors-label-dark-l: 50%;
+      --tw-colors-label-light-hs: 99, 44%;
+      --tw-colors-label-light-l: 50%;
+
+      --tw-colors-link-dark-hs: 129, 20%;
+      --tw-colors-link-dark-l: 50%;
+      --tw-colors-link-light-hs: 147, 83%;
+      --tw-colors-link-light-l: 50%;
+
+      --tw-colors-primary-dark-hs: 288, 33%;
+      --tw-colors-primary-dark-l: 50%;
+      --tw-colors-primary-light-hs: 210, 100%;
+      --tw-colors-primary-light-l: 50%;
+
+
+      --tw-colors-tertiary-dark-hs: 938, 22%;
+      --tw-colors-tertiary-dark-l: 50%;
+      --tw-colors-tertiary-light-hs: 233, 11%;
+      --tw-colors-tertiary-light-l: 50%;
+
+      --tw-colors-text-dark-hs: 0, 83%;
+      --tw-colors-text-dark-l: 50%;
+      --tw-colors-text-light-hs: 0, 88%;
+      --tw-colors-text-light-l: 50%;
+
+      --tw-colors-secondary-dark-hs: 0, 44%;
+      --tw-colors-secondary-dark-l: 50%;
+      --tw-colors-default-dark-hs: 234, 65%;
+      --tw-colors-default-dark-l: 50%;
+      --tw-colors-secondary-light-hs: 944, 77%;
+      --tw-colors-secondary-light-l: 50%;
+
+      --tw-colors-success-dark-hs: 231, 18%;
+      --tw-colors-success-dark-l: 50%;
+      --tw-colors-success-light-hs: 233, 13%;
+      --tw-colors-success-light-l: 50%;
+
+      --tw-colors-warning-dark-hs: 233, 9%;
+      --tw-colors-warning-dark-l: 50%;
+      --tw-colors-warning-light-hs: 455, 78%;
+      --tw-colors-warning-light-l: 50%;
+
+      /** font family*/
+      --tw-font-family-sans-sans-1: ;
+      --tw-font-size-sans-sans-2: ;
```

Comment:
I don't see these used in this PR, can you elaborate on their use? I also can't map these to design tokens configured within `tokens.cjs`, at least not how they are declared here.

Reply:
i declared becasue according to tokens.cjs line 68 it could be an option. Shall i remove it?
<img width="513" alt="image" src="https://github.com/ForgeRock/forgerock-web-login-framework/assets/18680830/e97ec8a7-0b7b-4056-a3f6-07ccf2efa180">

Reply:
Oh, I see. 

Just so you know, the array passed into `fontFamily.sans` is just an array of font families, but it's treated as a single value in CSS: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family, a comma delimited list of fonts. So, this is best defined as a single custom property that just takes a comma delimited list of values as seen in the CSS spec.

So, the custom property would look something like this:

```css
:root {
  --tw-font-family-sans: 'Open Sans', Calibri, Arial, Helvetica, sans-serif;
}
```